multicluster/controllers/multicluster/suite_test.go

```diff
@@ -86,7 +88,9 @@ func TestAPIs(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
-
+	//out, err := exec.Command("sudo", "kubectl", "get", "nodes").Output()
+	//logf.Log.Info("nodes","out", string(out), "err", err)
+	
 	By("bootstrapping test environment")
 	//TODO: KUBECONFIG should be set up by command line.
```

Comment:
ditto, since you already added steps to set KUBECONFIG, you can remove these comments.